lib/child_process.js

```diff
@@ -49,13 +49,15 @@ const {
   convertToValidSignal,
   getSystemErrorName
 } = require('internal/util');
+
```

Comment:
The whitespace changes seem unrelated?

Reply:
Yes, auto imports by vscode, fixed in 2b19cd5